build/media_source/plg_system_shortcut/js/keyselect.es6.js

```diff
@@ -0,0 +1,105 @@
+((document, Joomla) => {
+  'use strict';
+
+  if (!Joomla) {
+    throw new Error('Joomla API is not properly initialised');
+  }
+
+  let button = null;
+
+  const keyDown = (e) => {
+    if (e.keyCode >= 65 && e.keyCode <= 90) {
+      e.preventDefault();
+      e.stopPropagation();
+      e.stopImmediatePropagation();
+    }
+  };
+
+  const keyUp = (e) => {
+    if (e.keyCode >= 65 && e.keyCode <= 90) {
+      const pressedKeys = [];
+      if (e.ctrlKey) {
+        pressedKeys.push('CTRL');
+      }
+      if (e.shiftKey) {
+        pressedKeys.push('SHIFT');
+      }
+      if (navigator.platform.match('Mac') ? e.metaKey : e.altKey) {
+        pressedKeys.push('ALT');
+      }
+
+      pressedKeys.push(e.key.toUpperCase());
+
+      document.getElementById('newKeyCombination').value = pressedKeys.join(' + ');
+    }
+  };
+
+  const openModal = (e) => {
+    button = null;
+    if (!e.relatedTarget) {
+      return;
+    }
+
+    button = e.relatedTarget;
+    document.getElementById('currentKeyCombination').value = e.relatedTarget.innerText;
+  };
+
+  const closeDModal = () => {
+    document.getElementById('newKeyCombination').value = '';
+  };
+
+  const confirmKeys = () => {
+    const modal = document.getElementById('keySelectModal');
+    const modalInstance = bootstrap.Modal.getInstance(modal);
+    const value = Joomla.sanitizeHtml(document.getElementById('newKeyCombination').value);
+    if (button && value) {
+      button.innerText = value;
+      button.previousElementSibling.value = value;
+    }
+
+    modalInstance.hide();
+  };
+
+  const initialize = () => {
+    const modal = `
+      <div class="modal fade" id="keySelectModal" tabindex="-1" role="dialog" data-bs-backdrop="static" aria-labelledby="keySelectModalLabel" aria-hidden="true">
+        <div class="modal-dialog" role="document">
+          <div class="modal-content">
+            <div class="modal-header">
+              <h5 class="modal-title" id="keySelectModalLabel">${Joomla.Text._('PLG_SYSTEM_SHORTCUT_SET_SHORTCUT')}</h5>
```

Comment:
```suggestion
              <h3 class="modal-title" id="keySelectModalLabel">${Joomla.Text._('PLG_SYSTEM_SHORTCUT_SET_SHORTCUT')}</h3>
```

prvents triggering a warning about skipped headings and is consistent with similar modals